internal/function/blame.go

```diff
@@ -0,0 +1,145 @@
+package function
+
+import (
+	"fmt"
+	"github.com/src-d/gitbase"
+	"gopkg.in/src-d/go-git.v4"
+	"gopkg.in/src-d/go-mysql-server.v0/sql"
+
+	"gopkg.in/src-d/go-git.v4/plumbing"
+	"gopkg.in/src-d/go-git.v4/plumbing/object"
+)
+
+type (
+	// Blame implements git-blame function as UDF
+	Blame struct {
+		repo   sql.Expression
+		commit sql.Expression
+	}
+
+	// BlameLine represents each line of git blame's output
+	BlameLine struct {
+		Commit  string `json:"commit"`
+		File    string `json:"file"`
+		LineNum int    `json:"linenum"`
+		Author  string `json:"author"`
+		Text    string `json:"text"`
+	}
+)
+
+// NewBlame constructor
+func NewBlame(repo, commit sql.Expression) sql.Expression {
+	return &Blame{repo, commit}
+}
+
+func (b *Blame) String() string {
+	return fmt.Sprintf("blame(%s, %s)", b.repo, b.commit)
```

Comment:
Could you check if other function implementations are using upper case instead of lower?

Reply:
All are using lowercase except `IS_VENDOR`.